plugins/modules/cloud/misc/terraform.py

```diff
@@ -449,12 +474,67 @@ def main():
     if state == 'present' and module.params.get('parallelism') is not None:
         command.append('-parallelism=%d' % module.params.get('parallelism'))
 
-    variables_args = []
-    for k, v in variables.items():
-        variables_args.extend([
-            '-var',
-            '{0}={1}'.format(k, v)
-        ])
+    def process_args(variables, top):
```

Comment:
```suggestion
    def process_args(variables, top=True):
```
Then you don't need to explicitly pass `True` in the initial call.